app/src/main/java/org/kiwix/kiwixmobile/wifi_hotspot/WifiHotspotManager.java

```diff
@@ -0,0 +1,86 @@
+package org.kiwix.kiwixmobile.wifi_hotspot;
+
+import android.content.Context;
+import android.net.wifi.WifiConfiguration;
+import android.net.wifi.WifiManager;
+import android.os.Build;
+import android.os.Handler;
+import android.util.Log;
+import androidx.annotation.RequiresApi;
+import org.kiwix.kiwixmobile.webserver.ServerStateListener;
+
+/**
+ * WifiHotstopManager class makes use of the Android's WifiManager and WifiConfiguration class
+ * to implement the wifi hotspot feature.
+ * Created by Adeel Zafar on 28/5/2019.
+ */
+
+public class WifiHotspotManager {
+  private WifiManager wifiManager;
+  Context context;
+  WifiManager.LocalOnlyHotspotReservation hotspotReservation;
+  boolean oreoenabled;
```

Comment:
this name feels like it could really be updated or perhaps just removed

Reply:
Fixed ebeff0abeb5cf364c2bba1d2b0413733ccc77711

Reply:
Fixed. 2604fc9dbd6a8bb871e0d0b65680a40f2762066b